Source/core/maps/GoogleMapsManager.js

```diff
@@ -1,55 +1,90 @@
 /**
- * Created by Kolpa on 22.03.2016.
- * Contributed by PrateekGupta1509
+ * MapsManager for Google Maps
+ * @author Kolpa (Kolya Opahle)
+ * @author PrateekGupta1509 (Prateek Gupta)
+ * @author heychirag (Chirag Arora)
+ */
+
+/**
+ * GoogleMapsManager namespace.
+ * @constructor
+ * @param {object} footprintCore
+ * @param {object} settingsProvider
  */
 
 var GoogleMapsManager = function(footprintCore, settingsProvider) {
-    this.footprintCore = footprintCore;
-    this.settingsProvider = settingsProvider;
-    this.update();
+  this.footprintCore = footprintCore;
```

Comment:
In the future, please include code style improvements in separate PRs... Otherwise they make it harder for me to spot what has actually changed in the code.